subsys/net/l2/canbus/canbus.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(net_l2_canbus, LOG_LEVEL_NONE);
+
+#include <net/net_core.h>
+#include <net/net_l2.h>
+#include <net/net_if.h>
+#include <net/net_pkt.h>
+#include <net/socket_can.h>
+
+static inline enum net_verdict canbus_recv(struct net_if *iface,
+					   struct net_pkt *pkt)
+{
+	net_pkt_lladdr_src(pkt)->addr = NULL;
+	net_pkt_lladdr_src(pkt)->len = 0;
+	net_pkt_lladdr_src(pkt)->type = NET_LINK_DUMMY;
```

Comment:
So, why NET_LINK_CANBUS was introduced, if NET_LINK_DUMMY is used here anyway?

Reply:
Bug.